install/master-setup.sh

```diff
@@ -91,15 +91,20 @@ if [[ "${MASTER_ID}" != "" ]]; then
 fi
 
 # stop the Cluster-Insight minion collectors.
```

Comment:
Please change the comment to:
# stop the Cluster-Insight replication controller (and its minions) if it is running.